tpl/template_funcs.go

```diff
@@ -1132,6 +1134,35 @@ func highlight(in interface{}, lang, opts string) (template.HTML, error) {
 	return template.HTML(helpers.Highlight(html.UnescapeString(str), lang, opts)), nil
 }
 
+// execute a command and returns its output as string
```

Comment:
>> @@ -1132,6 +1134,35 @@ func highlight(in interface{}, lang, opts
> string) (template.HTML, error) {
>>      return
> template.HTML(helpers.Highlight(html.UnescapeString(str), lang,
> opts)), nil
>>  }
>> 
>> +// execute a command and returns its output as string
> 
> Use a proper godoc comment. Start with the func name and use complete
> sentences with punctuation.
> 
> // exec executes a command and returns its output as a string.

In my opinion, sentences starting with a small letter don't look right. 
In this case, I would rather like to write

```
// Function exec executes a command and returns its output as a string.
````

But as I haven't found this in other places I modified it as you 
suggested.

I've also implemented all the other suggestions. Thanks for your help.